libs/ui/src/theme/global/globalStyles.tsx

```diff
@@ -17,6 +17,13 @@ export const GlobalStyles = createGlobalStyle`
   -moz-osx-font-smoothing: grayscale; 
   text-rendering: optimizeLegibility;
 
+	#root {
```

Comment:
indent seems strange on this in the review panel but it's solid in the source. Strange

Reply:
Fixed